### PR TITLE
RangerFinder NRA15/24 from nanoradar

### PR DESCRIPTION
**Pull request by error414**
Rangerfinder from nanoradar NRA15/NRA24 http://en.nanoradar.cn/Article/detail/id/477.html

Works pretty nice. https://www.youtube.com/watch?v=Hrc_3wNdrnU

Wiki updated as well.

### CODE REVIEW

**Comment by DzikuVx** [comment]
@error414 let's resolve conflicts and have it merged

**Comment by error414** [comment]
rebased

File: src/main/fc/settings.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ tables:
     values: ["NONE", "AUTO", "MPU6000", "MPU6500", "MPU9250", "BMI160", "ICM20689", "BMI088", "ICM42605", "BMI270","LSM6DXX",  "FAKE"]
     enum: accelerationSensor_e
   - name: rangefinder_hardware
-    values: ["NONE", "SRF10", "VL53L0X", "MSP", "BENEWAKE", "VL53L1X", "US42", "TOF10120_I2C", "FAKE", "TERARANGER_EVO"]
```
Comment (mmosca) [inline src/main/fc/settings.yaml:8]: NRA15/NRA24 could be called just NRA or NRAXX

Comment (error414) [reply src/main/fc/settings.yaml:8]: I renamed it to "NRA", it should be OK now.

**Pull request merged**